Gems/OpenParticleSystem/gem.json

```diff
@@ -0,0 +1,18 @@
+{
+    "gem_name": "OpenParticleSystem",
+    "display_name": "OpenParticleSystem",
+    "icon_path": "preview.png",
+    "type": "Code",
+    "license": "Apache-2.0 Or MIT",
+    "license_url": "https://github.com/o3de/o3de/blob/development/LICENSE.txt",
+    "origin": "Open 3D Engine - o3de.org",
+    "origin_url": "https://github.com/o3de/o3de",
+    "summary": "The OpenParticleSystem Gem provides a particle system framework alongside a dedicated toolset for creating natural phenomena (such as flames, smoke, rainfall, and snowfall) and specialized effects (including explosions), leveraging vast numbers of micro-particle elements and dynamic motion simulations to achieve realistic visual representations.",
```

Comment:
Please add an extra sentence we will remove later.  `WARNING: This gem is in preview, file formats and data may change before official release.`